packages/@react-native-windows/tester/src/js/examples/View/ViewExample.windows.js

```diff
@@ -368,8 +377,8 @@ function LayoutConformanceBox(): React.Node {
     <View
       style={{
         backgroundColor: 'blue',
-        width: 100,
-        height: 100,
+        width: 60,
```

Comment:
Why are we forking this?

Reply:
Meta altered the testid's for the ViewExample page recently. I needed that state of the file so that we aren't diverging our testid's with what's upstream. We haven't ingested that commit via integrations yet, but to avoid having to redo the snapshots again when the integration arrives I ingested the change now.